src/Entity/Comment.php

```diff
@@ -126,12 +126,12 @@ public function setAuthor(User $author): void
         $this->author = $author;
     }
 
-    public function getPost(): Post
+    public function getPost(): ?Post
     {
         return $this->post;
     }
 
-    public function setPost(Post $post): void
+    public function setPost(?Post $post): void
```

Comment:
Btw, could we do the same in `Post::setTitle()` & Co. to fix https://github.com/symfony/demo/issues/741?

Reply:
Good idea! Done.